ui/src/app/components/home/home.component.scss

```diff
@@ -27,4 +27,7 @@
       border-left: 1px solid #fafafa;
     }
   }
+  button{
```

Comment:
Remove this local CSS. Instead add .mat-raised-button.mat-primary {
color: white;
background-color: #3367d6;
} in global.

Reply:
Made the suggested changes